packages/next-steps/index.tsx

```diff
@@ -0,0 +1,75 @@
+import Link from 'next/link'
+import classNames from 'classnames'
+import type { NextStepsProps } from './types'
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import useProductMeta from '@hashicorp/platform-product-meta'
+import Intro from '@hashicorp/react-intro'
+import s from './style.module.css'
+
+export default function NextSteps({
+  appearance = 'light',
+  theme = 'hashicorp',
+  heading,
+  description,
+  ctas,
+  steps,
```

Comment:
Potentially an improvement:

```js
if(ctas.length >= 3 && ctas.length % 2 == 1) {
  // make the first item full width
}
```

<img width="1442" alt="CleanShot 2022-04-20 at 09 36 35@2x" src="https://user-images.githubusercontent.com/2105067/164280145-68b27d9a-b1e1-42ca-a59d-d5752e94edde.png">

Reply:
Or rather, not `ctas` but `steps`!

Reply:
One of the limitations of Swingset is missing out on TS feedback. We do have the interface setup to only accept up to three steps https://github.com/hashicorp/react-components/pull/564/files#diff-583ce2f482e438c4109dab19ae8d9e87d2a3514bf4c7672bc563d2224510f062R46-R49

Reply:
Got it!  Okay then no longer applies 💯